frontend/packages/dev-console/integration-tests/features/customization/customization-of-catalog.feature

```diff
@@ -0,0 +1,40 @@
+@customize-catalogs
+Feature: Customization of catalogs
+                Allows cluster admins to selectively disable a sub-catalog and all links to the specific sub-catalog or disable the complete developer catalog
+
+        Background:
+            Given user has created or selected namespace "aut-developer-catalog"
+
+ @regression @manual
+        Scenario: When all the sub-catalogs are disabled: DC-01-TC01
+            Given user is at cluster YAML of "operator.openshift.io/v1" console
+             When user adds the "Add sub-catalog types" code snippet under spec.customization.developerCatalog.types
+              And user changes the state to "Disabled" and removes enabled attribute
+              And user clicks on Save button
+             Then user will not see "Developer Catalog" and all the sub-catalogs in Add page and Topology page
+              And user will not get any link to catalog page 
```

Comment:
This should be well defined i.e. define from where the catalog link will be disappeared

Reply:
Hi @sanketpathak , added the details.